SMDT1.py

```diff
@@ -0,0 +1,168 @@
+from Bio import SeqIO
+from Bio.SeqUtils import gc_fraction
+from typing import Union
+import numpy as np
+import os
+import string
+
+COMP_BASES_DNA: dict = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A', 'a': 't',
+                        'g': 'c', 'c': 'g', 't': 'a'}
+COMP_BASES_RNA: dict = {'A': 'U', 'C': 'G', 'G': 'C', 'U': 'A', 'a': 'u',
+                        'g': 'c', 'c': 'g', 'u': 'a'}
+AA_MASSES: dict = {'A': 89, 'R': 174, 'N': 132, 'D': 133, 'C': 121, 'Q': 146, 'E': 147, 'Z': 147,
+                   'G': 75, 'H': 155, 'I': 131, 'L': 131, 'K': 146, 'M': 149, 'F': 165, 'P': 115, 'S': 105,
+                   'T': 119, 'W': 204, 'Y': 181, 'V': 117}
+
+class BiologicalSequence:
+    def __init__(self, seq):
+        self.seq = seq
+
+    def __len__(self):
+        return len(self.seq)
+
+    def __str__(self):
+        return str(self.seq)
+
+    def __repr__(self):
+        return f"{self.seq}"
```

Comment:
как мне подсказали в моём ревью, возможно можно добавить доп. информацию о классе, например 
```suggestion
    def __repr__(self):
        return f'NucleicAcid("{self.sequence}")'
```